config/hubs/meom-ige.cluster.yaml

```diff
@@ -75,7 +75,7 @@ hubs:
                 description: "~32 CPU, ~128G RAM"
                 kubespawner_override:
                   mem_limit: 128G
-                  mem_guarantee: 110G
+                  mem_guarantee: 100G
                   node_selector:
                     node.kubernetes.io/instance-type: n1-standard-32
               - display_name: "Huge"
```

Comment:
is there something similar we should do for "huge"?

Reply:
Fetching info from the nodes:

```
small >> 5758056Ki >> 5.75 GB allocatable (mem_guarantee: 5GB)
medium  >> 27155328Ki >> 27.15 GB allocatable (mem_guarantee: 25GB)
large >> 56183024Ki >> 56.18 GB allocatable (mem_guarantee: 50GB)
very large >> 114336712Ki >> 114.33G allocatable (mem_guarantee: 100GB with this PR)
huge >> 235367296Ki >> 235.37GB allocatable (mem_guarantee: 220G)
```

I think we are OK with the huge one but happy to discuss all the other values 😉  .